src/common/cast/remote-player-manager.js

```diff
@@ -21,39 +24,49 @@ class RemotePlayerManager {
     }
   }
 
-  static load(castConfig: Object, remoteControl: RemoteControl): void {
+  load(castConfig: Object, player: KalturaPlayer): void {
+    ++RemotePlayerManager._loadedPlayerCount;
```

Comment:
use count per type

Reply:
what if a player destroyed, need to decrease the counter no?

Reply:
right now the remote players are only destroyed all at once and not one at a time

Reply:
so the question is whether one kaltura player on the page might be destroyed while other kaltura players are not destroyed

Reply:
sure. the app can destroy a specific instance.